webpack.js

```diff
@@ -0,0 +1,158 @@
+const webpackConfig = require('@nextcloud/webpack-vue-config')
+
+const fs = require('fs')
+const gettextParser = require('gettext-parser')
+const glob = require('glob')
+const md5 = require('md5')
+const path = require('path')
+
+const { DefinePlugin } = require('webpack')
+const BabelLoaderExcludeNodeModulesExcept = require('babel-loader-exclude-node-modules-except')
+const nodeExternals = require('webpack-node-externals')
+
+// scope variable
+// fallback for cypress testing
+const appVersion = JSON.stringify(process.env.npm_package_version || 'nextcloud-vue')
+const versionHash = md5(appVersion).substr(0, 7)
+const SCOPE_VERSION = JSON.stringify(versionHash)
+
+console.info('This build version hash is', versionHash, '\n')
+
+// https://github.com/alexanderwallin/node-gettext#usage
+// https://github.com/alexanderwallin/node-gettext#load-and-add-translations-from-mo-or-po-files
+const translations = fs
+	.readdirSync('./l10n')
+	.filter(name => name !== 'messages.pot' && name.endsWith('.pot'))
+	.map(file => {
+		const path = './l10n/' + file
+		const locale = file.substr(0, file.length - '.pot'.length)
+
+		const po = fs.readFileSync(path)
+		const json = gettextParser.po.parse(po)
+
+		// Compress translations Content
+		const translations = {}
+		for (const key in json.translations['']) {
+			if (key !== '') {
+				// Plural
+				if ('msgid_plural'in json.translations[''][key]) {
+					translations[json.translations[''][key].msgid] = {
+						pluralId: json.translations[''][key].msgid_plural,
+						msgstr: json.translations[''][key].msgstr,
+					}
+					continue
+				}
+
+				// Singular
+				translations[json.translations[''][key].msgid] = json.translations[''][key].msgstr[0]
+			}
+		}
+
+		return {
+			locale,
+			translations,
+		}
+	})
```

Comment:
Would be nice to put this into an external script file :)

Reply:
It's located in `translations.js` now.